backend/src/test/java/de/laura/project/controller/TriviaControllerTest.java

```diff
@@ -39,45 +45,104 @@ private String getUrl() {
+    }
+
+
+    @Test
+    @DisplayName("Checks whether user's answer is correct")
+
+    public void checkAnswerIsCorrect() {
+
+        //GIVEN
+        List<String> answerForTriviaQuestionSet = List.of("correct answer", "answerOne", "answerTwo", "answerThree");
+        List<String> anotherAnswerForTriviaQuestionSet = List.of("another correct answer", "answerOne", "answerTwo", "answerThree");
+
+        tempTriviaQuestionDB.getTriviaQuestionSetList().add(new TriviaQuestionSet(1, "This is an awesome question", "correct answer", answerForTriviaQuestionSet));
+        tempTriviaQuestionDB.getTriviaQuestionSetList().add(new TriviaQuestionSet(2, "This is another awesome question", "another correct answer", anotherAnswerForTriviaQuestionSet));
+
+        TriviaAnswerDTO triviaAnswerDTO = new TriviaAnswerDTO(1, "correct answer");
+
+        //WHEN
+        ResponseEntity<Boolean> response = testRestTemplate.postForEntity(getUrl() + "/answer", triviaAnswerDTO, Boolean.class);
+
+        //THEN
+        assertThat(response.getStatusCode(), is(HttpStatus.OK));
+        assertTrue(response.getBody());
+    }
+
+    @Test
+    @DisplayName("Checks whether user's incorrect answer returns false")
+
+    public void checkAnswerIsFalse() {
+
+        //GIVEN
+        List<String> answerForTriviaQuestionSet = List.of("correct answer", "answerOne", "answerTwo", "answerThree");
+        List<String> anotherAnswerForTriviaQuestionSet = List.of("another correct answer", "answerOne", "answerTwo", "answerThree");
+
+        tempTriviaQuestionDB.getTriviaQuestionSetList().add(new TriviaQuestionSet(1, "This is an awesome question", "correct answer", answerForTriviaQuestionSet));
+        tempTriviaQuestionDB.getTriviaQuestionSetList().add(new TriviaQuestionSet(2, "This is another awesome question", "another correct answer", anotherAnswerForTriviaQuestionSet));
+
+        TriviaAnswerDTO triviaAnswerDTO = new TriviaAnswerDTO(1, "answerOne");
+
+        //WHEN
+        ResponseEntity<Boolean> response = testRestTemplate.postForEntity(getUrl() + "/answer", triviaAnswerDTO, Boolean.class);
+
+        //THEN
+        assertThat(response.getStatusCode(), is(HttpStatus.OK));
+        assertFalse(response.getBody());
+    }
+
+
+    @Test
+    @DisplayName("Calls question based on questionsID")
+
+    public void getSingleQuestion() {
+
+        //GIVEN
+        List<String> answerForTriviaQuestionSet = List.of("correct answer", "answerOne", "answerTwo", "answerThree");
+        List<String> anotherAnswerForTriviaQuestionSet = List.of("another correct answer", "answerOne", "answerTwo", "answerThree");
+
+        tempTriviaQuestionDB.getTriviaQuestionSetList().add(new TriviaQuestionSet(1, "This is an awesome question", "correct answer", answerForTriviaQuestionSet));
+        tempTriviaQuestionDB.getTriviaQuestionSetList().add(new TriviaQuestionSet(2, "This is another awesome question", "another correct answer", anotherAnswerForTriviaQuestionSet));
```

Comment:
if you use this code repeatedly in your tests you may want to consider extracting it into a private helper method and simply call that method each time saving you some code